internal/tfengine/schema.go

```diff
@@ -53,6 +53,32 @@ properties:
         data:
           descripton: Key value pairs passed to this template.
           type: object
+          properties:
```

Comment:
We could make this a mega schema, but I was planning on adding a schema for each recipe to keep this more manageable, else you will have every possible field here which will be confusing.

e.g. foundation.yamlschema, etc.

You can add a TODO to move this or just add this when the per-recipe schema is done.